app/__init__.py

```diff
@@ -510,8 +510,8 @@ def setup_blueprints(application):
     from app.main import no_cookie as no_cookie_blueprint
     from app.status import status as status_blueprint
 
-    main_blueprint.before_request(make_session_permanent)
-    main_blueprint.after_request(save_service_or_org_after_request)
+    # main_blueprint.before_request(make_session_permanent)
+    # main_blueprint.after_request(save_service_or_org_after_request)
```

Comment:
Remove rather than comment?  Also do you want to remove the function definitions as well since you have mow moved them into the blueprint?